doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,124 @@
+
+#################
+The Gen3 workflow
+#################
+
+These directions are for users who wish to create their own data repositories independent of any Rubin/LSST-supported shared repositories. They assume the user has a recent weekly version of ``lsst_distrib`` setup.
+
+Local repository setup
+----------------------
+
+Create a Gen3 data repository and write curated calibration products to it (these include crosstalk, linearity, and defects).
+The latter step creates the ``DECam/calib`` collection.
+In the example here, ``REPO`` refers to the filepath you choose for the Butler repository.
+
+.. prompt:: bash
+
+   cd REPO
+   butler create REPO
+   butler register-instrument REPO lsst.obs.decam.DarkEnergyCamera
+   butler write-curated-calibrations REPO lsst.obs.decam.DarkEnergyCamera
+
+Check that this worked successfully by querying all the collections currently in the repo, e.g.,
+
+.. prompt:: bash
+
+   butler query-collections REPO --chains=tree
+
+Ingest raw images
+-----------------
+
+Ingest raw science and calibration (bias and flat) frames, which creates the ``DECam/raw/all`` collection.
+Next, run overscan correction on all the raws via the ``RunIsrForCrosstalkSources.yaml`` pipeline so they may be used as crosstalk sources during ISR (this is a DECam-specific step).
+
+.. note::
+   Until `DM-30651 <https://jira.lsstcorp.org/browse/DM-30651>`__ is resolved, consider including the following configurations during the overscan crosstalk prep step.
+
+   When running the ``RunIsrForCrosstalkSources.yaml`` pipeline, append ``-c overscan:overscan.fitType='MEDIAN_PER_ROW'``.
+
+.. prompt:: bash
+
+   butler ingest-raws REPO /path/to/raw/science/*.fits.fz --transfer link
+   butler ingest-raws REPO /path/to/raw/calibs/*.fits.fz --transfer link
+   pipetask run -b REPO -i DECam/raw/all -o DECam/raw/crosstalk-sources -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml --register-dataset-types
+
+Check that this worked successfully, e.g.,
+
+.. prompt:: bash
+
+   butler query-dimension-records REPO exposure --where "instrument='DECam' AND exposure.observation_type='zero'"
+   butler query-dimension-records REPO exposure --where "instrument='DECam' AND exposure.observation_type='dome flat'"
+
+Use `~lsst.cp.pipe`_ to build nightly calibs
+--------------------------------------------
+
+Next, build nightly (or similar) bias and flat frames and certify them into a calib collection.
+
+This example assumes the user has a single night of observations with bias frames numbered 1-6 and flat frames numbered 7-12, and wants to create nightly calib products valid for 24 hours.
+Note the bias building pipeline is camera-agnostic, but the flat-building pipeline has a prerequisite DECam-specific step which correctly handles inter-chip crosstalk.
+
+.. note::
+   Until `DM-30651 <https://jira.lsstcorp.org/browse/DM-30651>`__ is resolved, consider including the following configurations during bias and flat building.
```

Comment:
Worth noting on that ticket to remove these `note::`s as part of closing that ticket?